doc/source/appendix/citing-qibo.rst

```diff
@@ -0,0 +1,91 @@
+Publications
+============
+
+If Qibo has been significant in your research, and you would like to acknowledge
+the project in your academic publication, we suggest citing the following documents:
+
+Peer-Reviewed Articles
+----------------------
+
+* S. Efthymiou, S. Ramos-Calderer, C. Bravo-Prieto, A. Pérez-Salinas, D.
+  Garcı́a-Martı́n, A. Garcia-Saez, J. I. Latorre, S. Carrazza, *Qibo: a
+  framework for quantum simulation with hardware acceleration*, Quantum Science
+  and Technology 7 (1) (2021) 015018. `doi:10.1088/2058-9565/ac39f5`_,
+  (`arXiv:2009.01845`_).
+
+  In *BibTeX* format:
+
+  .. code-block:: text
+
+    @article{qibo_paper,
+        doi       = {10.1088/2058-9565/ac39f5},
+        url       = {https://doi.org/10.1088/2058-9565/ac39f5},
+        year      = 2021,
+        month     = {dec},
+        publisher = {{IOP} Publishing},
+        volume    = {7},
+        number    = {1},
+        pages     = {015018},
+        author    = {Stavros Efthymiou and
+                     Sergi Ramos-Calderer and
+                     Carlos Bravo-Prieto and
+                     Adri{\'{a}}n P{\'{e}}rez-Salinas and
+                     Diego Garc{\'{\i}}a-Mart{\'{\i}}n and
+                     Artur Garcia-Saez and
+                     Jos{\'{e}} Ignacio Latorre and
+                     Stefano Carrazza},
+        title     = {Qibo: a framework for quantum simulation with hardware acceleration},
+        journal   = {Quantum Science and Technology},
+    }
+
+.. _`doi:10.1088/2058-9565/ac39f5`: https://doi.org/10.1088/2058-9565/ac39f5
+.. _`arXiv:2009.01845`: https://arxiv.org/abs/2009.01845
+
+Software References in Zenodo
+-----------------------------
+
+* S. Efthymiou and S. Carrazza. (2021). qiboteam/qibojit: qibojit. Zenodo.
+  `https://doi.org/10.5281/zenodo.5248470`_.
+
+  In *BibTeX* format:
+
+  .. code-block:: text
+
+    @software{qibojit_zenodo,
+        author    = {Stavros Efthymiou and Stefano Carrazza},
+        title     = {qiboteam/qibojit},
+        month     = aug,
+        year      = 2021,
+        publisher = {Zenodo},
+        doi       = {10.5281/zenodo.5248470},
+        url       = {https://doi.org/10.5281/zenodo.5248470}
+    }
+
+.. _`https://doi.org/10.5281/zenodo.5248470`: https://doi.org/10.5281/zenodo.5248470
+
+* S. Efthymiou, S. Ramos-Calderer, C. Bravo-Prieto, A.
+  Pérez-Salinas, D. García-Martín, A. Garcia-Saez, J. I. Latorre, S. Carrazza.
+  (2020). qiboteam/qibo: Qibo. Zenodo. `https://doi.org/10.5281/zenodo.3997194`_.
+
+  In *BibTeX* format:
+
+  .. code-block:: text
+
+    @software{qibo_zenodo,
+        author    = {Stavros Efthymiou and
+                     Sergi Ramos-Calderer and
+                     Carlos Bravo-Prieto and
+                     Adri{\'{a}}n P{\'{e}}rez-Salinas and
+                     Diego Garc{\'{\i}}a-Mart{\'{\i}}n and
+                     Artur Garcia-Saez and
+                     Jos{\'{e}} Ignacio Latorre and
+                     Stefano Carrazza},
```

Comment:
Not sure if it matters but the author list here (and also some author names) does not match the one [exported from zenodo](https://zenodo.org/record/5711842/export/hx#.YghsPHVByV4). I guess zenodo is updated with each qibo version? Then perhaps it may be better to remove the bibtex format from here and leave only the links, or somehow link what appears here with the latest version from zenodo.

Reply:
Indeed, thanks, makes sense.